sumo-make build-setup fails on Vagrant environment

<!--
Thanks for filing an issue!

If you notice any issue with Collection, please see our troubleshooting guide:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/docs/Troubleshoot_Collection.md

If you did not find a solution in troubleshooting guide please check that we do not have any issue describing your case already open.

If you think that your issue is a new bug, please add as much context as you can using the template below but feel free to adjust it.
-->

**Describe the bug**

`sumo-make build-setup` fails on Vagrant environment

**Logs**

```
sumo-make build-setup
cd /sumologic/ && /usr/bin/make -C /sumologic/deploy/docker/setup build
make[1]: *** /sumologic/deploy/docker/setup: No such file or directory.  Stop.
make: *** [/sumologic/vagrant/Makefile:22: build-setup] Error 2
```

**To Reproduce**

In clean repository follow steps described in https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/vagrant

**Environment (please complete the following information):**
 - **Collection version** :  commit: 490e2c5379b7d0956dfd88755639529d1cfeca3f

**Anything else do we need to know**
`/sumologic/deploy/docker/setup` was removed in https://github.com/SumoLogic/sumologic-kubernetes-collection/commit/a417109f36b34dbe9e0444d55786d8cf42a0617b#diff-b7bd55c11b781b0ccc43aa6e57f9dadf0660e9d1d4e27e0979ee43a407d454ae

--- a/vagrant/Makefile
+++ b/vagrant/Makefile
@@ -18,19 +18,6 @@ namespace = sumologic
 test:
 	$(MAKE) -C ${root_dir} test
 
-build-setup:
-	cd ${root_dir} && $(MAKE) -C ${root_dir}deploy/docker/setup build
-	${mkfile_path} \
-		tag-and-push-local-setup
-
-tag-and-push-local-setup:
-	$(eval timestamp := $(shell date +%s))
-	docker tag kubernetes-setup:latest localhost:32000/kubernetes-setup:local-${timestamp}
-	docker push localhost:32000/kubernetes-setup:local-${timestamp}
-	touch ${local_values_file}
-	yq w -i ${local_values_file} sumologic.setup.job.image.repository localhost:32000/kubernetes-setup
-	yq w -i ${local_values_file} sumologic.setup.job.image.tag local-${timestamp}
-
 generate-local-config:
 	${vagrant_scripts_dir}/generate-local-config.sh
 

--- a/vagrant/README.md
+++ b/vagrant/README.md
@@ -32,20 +32,6 @@ vagrant ssh
 
 NOTICE: The directory with sumo-kubernetes-collection repository on the host is synced with `/sumologic/` directory on the virtual machine.
 
-## Build setup job
-
-On the virtual machine, to build the setup job Docker image please use `build-setup` target:
-
-```bash
-sumo-make build-setup
-```
-
-or
-
-```bash
-/sumologic/vagrant/Makefile build-setup
-```
-
 ## Collector
 
 To install or upgrade collector please type:

